README.md

```diff
@@ -5,10 +5,14 @@ Respond to short response questions in clear, concise sentences directly within
 
 ### Short Response Questions
 **1. What is recursion?**
+Answer: Recursion is a method or process of breaking down problems into smaller and smaller subproblems. You continue to do this until it gets small enough that it can be solved with ease. 
```

Comment:
:100: Perfect! Be sure to mention that this is possible by having a function call itself